.github/workflows/release.yml

```diff
@@ -26,17 +26,87 @@ jobs:
         run: yarn package
       - name: integration tests
         run: test/run-against-dist test/test-all.sh
+      - name: Upload artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: dist
+          path: dist
 
-      # publish to package managers only if this is a new version      
-      - run: yarn release
+  release_maven:
+    name: Release to Maven
+    needs: build_artifact
+    runs-on: ubuntu-latest
+    container:
+      image: jsii/superchain
+    steps:
+      - name: Download build artifacts
+        uses: actions/download-artifact@v1
+        with:
+          name: dist
+      - name: Release
+        run: yarn release-maven
```

Comment:
How does this work? We didn't checkout the repo nor `yarn install` any dependencies? Am I missing something?

Reply:
Great catch :) I will just `npx` the JSII release scripts, and remove them from package.json